improver/between_thresholds.py

```diff
@@ -136,9 +136,11 @@ def _get_multiplier(self):
         relative_to_threshold = self.thresh_coord.attributes[
             "spp__relative_to_threshold"
         ]
-        if relative_to_threshold == "above":
+        if (relative_to_threshold in ("above", "greater_than",
+                                      "greater_than_or_equal_to")):
             multiplier = 1.0
-        elif relative_to_threshold == "below":
+        elif (relative_to_threshold in ("below", "less_than",
+                                        "less_than_or_equal_to")):
```

Comment:
Please could you use your new `above_or_below` function here.

Reply:
I've had a look at using the above_or_below function but it needs a bit of rejigging the code because of the way that the code is written. It isn't just trivial to swap it out (because of error messages and places where there may not be spp__relative_to_threshold attribute). I also think it's a bit easier to see like this what is going on?